backend/controller/console/console.go

```diff
@@ -67,9 +67,12 @@ func verbSchemaString(sch *schema.Schema, verb *schema.Verb) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	err = visitNode(sch, verb.Response, &verbString)
```

Comment:
I think this needs to be much smarter than this to really fix it, but this works for now. ie. transitive types will be duplicated.